homeassistant/components/isy994/cover.py

```diff
@@ -49,16 +65,30 @@ def is_closed(self) -> bool:
             return None
         return self._node.status == 0
 
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
+
     def open_cover(self, **kwargs) -> None:
         """Send the open cover command to the ISY994 cover device."""
-        if not self._node.turn_on(val=100):
+        val = 100 if self._node.uom == UOM_BARRIER else None
+        if not self._node.turn_on(val=val):
             _LOGGER.error("Unable to open the cover")
 
     def close_cover(self, **kwargs) -> None:
         """Send the close cover command to the ISY994 cover device."""
         if not self._node.turn_off():
             _LOGGER.error("Unable to close the cover")
 
+    def set_cover_position(self, **kwargs):
+        """Move the cover to a specific position."""
+        position = kwargs.get(ATTR_POSITION)
```

Comment:
`ATTR_POSITION` is validated via the service schema to always be present in the kwargs. We can do `kwargs[ATTR_POSITION]` instead.

Reply:
Picked up in #36454